src/TypeScript/Sarif.SDK/lib/AutoGenerated/sarif-types.ts

```diff
@@ -309,6 +312,10 @@ export interface Run {
   logicalLocations?: {
     [k: string]: LogicalLocation;
   };
+  /**
+   * An array of one or more unique 'graph' objects.
+   */
+  graphs?: Graph[];
```

Comment:
you'll want to regenerate after someone reviews my open change to update SARIF schema to absolute current.